src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -197,4 +197,7 @@ protected void doRun() throws Exception {
     }
   }
 
+  public static boolean hasCredentials() {
+    return !StringUtils.isEmpty(GlobalPluginConfiguration.get().getCredentialsId());
```

Comment:
@oatmealraisin or we could just update this check to check all the places we might be getting credentials from instead of just the plugin config.